docs/contributor_guide/developer_call_notes/2020_10_08.rst

```diff
@@ -0,0 +1,46 @@
+Developer Call 2020-10-08
+-------------------------
+
+Scribbler: `Richard Edgar <https://github.com/riedgar-ms>`_
+
+Attendees:
+
+- `Miro Dudik <https://github.com/MiroDudik>`_
+- `Richard Edgar <https://github.com/riedgar-ms>`_
+- `Adrin Jalali <https://github.com/adrinjalali>`_
+- `Diondra Peck <https://github.com/diondrapeck>`_
+- Alex Quach
+- `Kevin Robinson <https://github.com/kevinrobinson>`_
+- `Mehrnoosh Sameki <https://github.com/mesameki>`_
+- `Hanna Wallach <https://www.microsoft.com/en-us/research/people/wallach/>`_
+- `Hilde Weerts <https://github.com/hildeweerts>`_
+
+Agenda:
+
+- Requirements for next release
+    - Still need new metrics API
+    - Need to announce deprecation of ``FairnessDashboard``
+    - Need to make sure Quickstart will actually work
+      (short term fix: put note about running `pip install` against GitHub at the top)
+    - Not clear what ``ThresholdOptimizer`` doc fixes Roman had in mind
+      but they probably shouldn't block the release
```

Comment:
@riedgar-ms re UI: before the meeting, I had thought that 0.5 would remove the `FairnessDashboard` and include matplotlib functions instead (eg, we'd ship https://github.com/fairlearn/fairlearn/pull/561 and some smaller follow-ons in coordination with the metrics API changes).

During the meeting, I came to understand that for 0.5 we'd:
- include the 0.4.6 version of the`FairnessDashboard` but adapted to Flask
- drop the post-0.4.6 UI code, which was never deployed
- punt on https://github.com/fairlearn/fairlearn/pull/561#discussion_r501847696  and matplotlib charts replacements for `FairnessDashboard` until the next release

This all seems fine to me; I'm not disagreeing with anything :)  But can you confirm I took away the right understanding?  Thanks!

Reply:
Obviously I missed the call yesterday, but I personally don't see why we have to wait with the matplotlib visualizations. It's fine if they're already in the package in the version where we're telling users that the `FairlearnDashboard` is going away, isn't it?

Other than that your description is in line with my mental model.

@riedgar-ms can you clarify please.

Reply:
@romanlutz My impression is that since it's been so long since a release, there's already a lot of major changes batched up on the main branch, the intention is to limit scope.  fwiw this seems super pragmatic and fine to me 👍

Reply:
Whatever works for you will be fine.

Tangentially related: we should move to a more frequent release rhythm, perhaps once a week or every other week.